src/login/logind-user.c

```diff
@@ -891,7 +893,17 @@ int config_parse_user_tasks_max(
         assert(rvalue);
         assert(data);
 
-        /* First, try to parse as percentage */
+        if (isempty(rvalue)) {
+                *m = manager->user_tasks_max;
```

Comment:
Can you please elaborate? "UserTasksMax=" seems to work correctly with the above code here. What am I missing?